content/en/docs/reference/command-line-tools-reference/feature-gates/ImageVolume.md

```diff
@@ -9,6 +9,9 @@ stages:
   - stage: alpha
     defaultValue: false
     fromVersion: "1.31"
```

Comment:
```suggestion
    fromVersion: "1.31"
    toVersion: "1.32"
```

Reply:
@saschagrunert, you also have to mention `toversion` which represents the version until which the feature gate is available.

Reply:
Sounds good, thank you. Fixed as suggested.